default/stringtables/en.txt

```diff
@@ -4219,6 +4219,39 @@ Next are icons showing the player empire total [[encyclopedia INDUSTRY_TITLE]],
 
 On the right hand side of the control bar are a number of icons for toggling display of other significant UI windows. Tooltips are available for these.'''
 
+DESIGN_WINDOW_ARTICLE_TITLE
+Design Window
+DESIGN_WINDOW_ARTICLE_TEXT
+'''The design window is for viewing or altering blueprints of monsters or ships. 
```

Comment:
I'm trying to create a difference in meaning between blueprints as the plan for a ship, the queued production of a ship and the ship which comes off the end of the production line.  

It is to get ahead of the question, "Why doesn't changing the design change all of the already queued designs as well?".  If people are asking themselves, "Why doesn't changing the blueprint, change the design in the queue?", then I was hoping the answer would be clear.

The question was asked [here.](http://freeorion.org/forum/viewtopic.php?f=25&t=10090&sid=d961978af4ed03a5c22e0d7bcad1a375)  I think that it is a question that will keep arising unless the UI is more obvious in the way it works.

So, I could either remove the word blueprint, or go all in and make the design window the blueprint window?